jboss-eap-standalone-rhel7/README.md

```diff
@@ -146,13 +147,10 @@ This section is to give you an idea on troubleshooting the issues that you might
 
     `more jbosseap.install.log`
 
-If you select BYOS as RHEL OS License type, then please make sure that you complete some pre-requisite steps before you start the deployment. To provision the RHEL-BYOS VM in your subscription, you will have to enable it in the Cloud Access from Red Hat portal and activate Red Hat Gold Images for your subscription. You can enable subscription for cloud access by following the instructions mentioned [here](https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/red_hat_cloud_access_reference_guide/con-enable-subs) and activate the Red Hat Gold Images by following the instructions mentioned [here](https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/red_hat_cloud_access_reference_guide/using_red_hat_gold_images#con-azure-access). If this is not done you will face issue with validating the template due to marketplace purchase eligibility failed.
-
-Once your Azure subscription is enabled, please follow this [link](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/byos) to accept the Marketplace terms for RHEL-BYOS image from your Azure subscription otherwise your template will fail.
 
 ## Notes
 
-If you don't have a Red Hat subscription to install a JBoss EAP, you can go through WildFly instead of JBoss EAP:
+If you don't have a Red Hat subscription to install JBoss EAP, you can deploy through WildFly instead of JBoss EAP:
```

Comment:
We have removed “Notes” section in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know whether we must add it